### DISCUSSION TITLE
NWEA has a few front end openings

### DISCUSSION THREAD

**Discussion by corinnaSchultz** [Nithur-M/work-from-anywhere#12] [Q&A]
It's tough to find good people, so maybe we could benefit from being included in your list.  
(I do not speak for NWEA, I'm just a cog in the machine, always on the lookout for good coworkers.)

https://jobs.jobvite.com/nwea/job/oPLEifwg
https://jobs.jobvite.com/nwea/job/oGjdkfwg

NWEA is an educational nonprofit based in the US. We advocate for growth-based assessments that actually provide value to students and teachers, including students who use assistive technology.

**Comment by Nithur-M** [comment]
Hi, your postings look interesting. I'll charge some amount to post your ads here because free sources like this can easily become spam if there are no charges and the quality will drastically degrade. I want to avoid that. Is it ok with you? I am happy to connect via email or Twitter. What do you say?

**Reply by corinnaSchultz** [reply]
I thought you were just looking for opportunities to post.  I'm just a random person on the internet, and it wasn't my intention to engage in anything commercial.

**Reply by Nithur-M** [reply]
Yes, I am looking to post opportunities and at the same time I should deal with potential spams.
